src/Common/ErrorCodes.cpp

```diff
@@ -614,6 +614,8 @@
     M(643, CANNOT_UNPACK_ARCHIVE) \
     M(644, REMOTE_FS_OBJECT_CACHE_ERROR) \
     \
+    M(700, INVALID_TRANSACTION) \
+    M(701, SERIALIZATION_ERROR) \
```

Comment:
This is to avoid resolving conflicts each time new error code is added, I will change 700/701 to 645/646 or similar before merging